yb-voyager/src/srcdb/yugabytedb.go

```diff
@@ -689,16 +794,15 @@ func (yb *YugabyteDB) filterUnsupportedUserDefinedDatatypes(tableName sqlname.Na
 			log.Warnf("close rows for query %q: %v", query, closeErr)
 		}
 	}()
+
 	var userDefinedDataTypes []string
 	for rows.Next() {
-		var dataType, isUserDefined string
-		err = rows.Scan(&dataType, &isUserDefined)
+		var qualifiedTypeName string
```

Comment:
Since you have changed the datatype from unqualified to qualified, can you confirm if the below check of unsupported types is working as expected?

Reply:
Or alternatively you can return typename only since you are already filtering the types that are present in the schemas of the table list only.

Reply:
> so even if there are two UDTs with same typename in two diffferent schemas that is okay because it doesn't, we just need whether the table type is a UDT, right?

I think you are mostly right but what if someone defines a UDT with name same as some native type.
Not sure if this is possible but same case like this For eg: custom_schema.int or custom_schema.varchar

It might end up detecting native cols as unsupported.

Reply:
> Not sure if this is possible but same case like this For eg: custom_schema.int or custom_schema.varchar
>It might end up detecting native cols as unsupported.

Even if that's a possible case - I don't think we were anyways not handling that earlier also so I guess since this is just an optimization PR we can keep it same and handle that separately (but we can file a bug if this is possible scenario and we have some false positives here) .

Reply:
@priyanshi-yb Yes we were not handling that and i have already fixed that bug in this PR itself.
It's a very minor thing and no one reported so i will skip ticket for this and add in the commit message.